### ISSUE TITLE
IPV6 on target other than iOS

### ISSUE THREAD

**Issue by starburst997** [openfl/lime#972]
When using an IPV6 network on target other than iOS, I get "Error: 6" while using the URLLoader.

To test IPV6 I setup a network using [Mac Internet Sharing](https://developer.apple.com/library/content/documentation/NetworkingInternetWeb/Conceptual/NetworkingOverview/UnderstandingandPreparingfortheIPv6Transition/UnderstandingandPreparingfortheIPv6Transition.html#//apple_ref/doc/uid/TP40010220-CH213-SW16)

I created a repo to do some test

https://github.com/starburst997/haxe-openfl-ipv6-ssl

***My results are as follow (with applying PR #6142):***

- [IPV6](https://github.com/starburst997/haxe-openfl-ipv6-ssl/tree/master/results/IPV6_PR6142.md)
- [IPV4](https://github.com/starburst997/haxe-openfl-ipv6-ssl/tree/master/results/IPV4_PR6142.md)

*Basically*:

```
URLLoader IO_ERROR: [IOErrorEvent type="ioError" bubbles=true cancelable=false text="6" errorID=0]
```

But works on iOS.

**Comment by jgranick** [comment]
I just did testing using the latest Lime development version and macOS, it is working properly, but it appears that there is a possibility that IPv6 would not have been enabled for Windows, so I've corrected that
